### ISSUE TITLE
npm voxel-registry and npm voxel-engine is incompatible

### ISSUE THREAD

**Issue by jlarsson** [voxel/voxel-registry#4]
*I apologize in advance if I reported this as an issue in the wrong context. This could technically be an issue with voxel-engine or my project setup.*

## Problem
Cant instantiate voxel-engine (npm version ~0.20.1) since it depends on voxel-registry (npm version ~0.6.0) which together with voxel-mesher introduces circular plugin dependencies.

## Solution
https://github.com/deathcap/voxel-registry/commit/3afdf7b6ae7cf197302a72861f5db9cd2a4bb726 probably fixes the problem, by removing 

```module.exports.pluginInfo = {
  loadAfter: ['voxel-mesher'] // optional, indicates opaque bit needed on non-transparent blocks (packed index)
};```

Still, commits after this is not reflected to npm and other modules, such as voxel-engine, would also require a dependency update. 

## Additional info
My package json is
```
{
  "scripts": {
    "start": "beefy index.js:bundle.js 99"
  },
  "dependencies": {
    "voxel-engine": "~0.20.1",
    "voxel-land": "~0.2.1",
    "voxel-plugins-ui": "~0.2.1"
  },
  "devDependencies": {
    "beefy": "~2.1.0",
    "browserify": "~5.10.0"
  }
  ...
}
```
The dependencies managed in voxel-engine are (my dump, not literal content in code)
```
"voxel-engine": [],
"voxel-registry": ["voxel-engine","voxel-mesher"],
"voxel-mesher": ["voxel-engine","voxel-registry","voxel-stitch"],
"voxel-stitch": ["voxel-engine","voxel-registry"],
"voxel-shader": ["voxel-engine","voxel-stitch","game-shell-fps-camera"],
"game-shell-fps-camera": ["voxel-engine"]
```
One can clearly see the circularity between voxel-registry and voxel-mesher.

**Comment by deathcap** [comment]
https://github.com/voxel/voxel-registry/commit/3afdf7b6ae7cf197302a72861f5db9cd2a4bb726 has since been published in voxel-registry 0.7.0+

**Comment by deathcap** [comment]
[voxel-engine-stackgl](https://github.com/voxel/voxel-engine-stackgl) 1.0.0 has since been published, with a voxel-registry ^0.8.1 dependency, so I believe this issue should be resolved @jlarsson if not let me know
